source/includes/_google_data_studio.md

```diff
@@ -0,0 +1,13 @@
+# LOCALiQ Data Studio Connector
```

Comment:
In my opinion, the LOCALiQ is redundant here as everything is localIq.  I suggest we just remove the word LOCALiQ.